assets/js/state/hosts.test.js

```diff
@@ -83,4 +85,27 @@ describe('Hosts reducer', () => {
 
     expect(hostsReducer(initialState, action)).toEqual(expectedState);
   });
+
+  it('should update the check selection for a host', () => {
+    const initialCheckSelection = [
+      faker.datatype.uuid(),
+      faker.datatype.uuid(),
+    ];
+    const host1 = hostFactory.build({ selected_checks: initialCheckSelection });
+    const host2 = hostFactory.build();
+    const initialState = { hosts: [host1, host2] };
+
+    const newChecksSelection = ['ABABAB', 'CDCDCD'];
```

Comment:
Maybe we should use `uuid` here, to match the format with the initial check selection